zepben-extensions/src/rmqpush.c

```diff
@@ -295,9 +298,234 @@ void send_voltage_report(struct TVoltageReport vr) {
     hv.minbus = (char*)vr.lv.min_bus;
     hv.maxbus = (char*)vr.lv.max_bus;
 
-    EnergyMeterReport emr = ENERGY_METER_REPORT__INIT;
-    emr.report_case = ENERGY_METER_REPORT__REPORT_VR;
-    emr.vr = &report;
+    OpenDssReport queueMsg = OPEN_DSS_REPORT__INIT;
+    queueMsg.report_case = OPEN_DSS_REPORT__REPORT_VR;
+    queueMsg.vr = &report;
     
-    send_energy_meter_report(&emr);
-}
\ No newline at end of file
+    send_opendss_message(&queueMsg);
+}
+
+void send_summary_report(struct TSummaryReport sr) {
+    SummaryReport report = SUMMARY_REPORT__INIT;
+
+    report.timestamp = time(NULL);
```

Comment:
question: Does the current time add any value to this report?

Reply:
timestamp removed after conversation on the #dev and other PR